hs_composite_resource/models.py

```diff
@@ -265,6 +265,18 @@ def delete_old_xml_files(folder=''):
                 # action is needed
                 pass
 
+    def is_aggregation_xml_file(self, file_path):
+        if not (file_path.endswith('_meta.xml') or file_path.endswith('_resmap.xml')):
+            return False
+
+        for logical_file in self.logical_files:
```

Comment:
Can make this considerably faster for resources with lots of files via 
```
for logical_file in self.logical_files.filter(file_path__endswith='.xml'): 
```
or even 
```
lf = self.logical_files.filter(Q(metadata_file_path=file_path) | Q(map_file_path=file_path))
    if lf.count(): 
       return True
    else: 
       return False
```

Reply:
I don't think these types of queries are possible since ```metadata_file_path``` and ```map_file_path``` are NOT model fields rather these are defined as properties of the model.

Reply:
@pkdash OK, no problem.